.idea/deploymentTargetSelector.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="deploymentTargetSelector">
+    <selectionStates>
+      <SelectionState runConfigName="app">
+        <option name="selectionMode" value="DROPDOWN" />
+      </SelectionState>
+    </selectionStates>
+  </component>
+</project>
```

Comment:
You can ignore the whole `/.idea`. Please add it to your [.gitignore](https://github.com/LoaiGamal/Stories-App/blob/main/.gitignore)